src/main/java/org/jenkinsci/deprecatedusage/Options.java

```diff
@@ -16,36 +24,53 @@
 
     private static final Options OPTIONS = new Options();
     private static List<String> additionalClasses;
+    private static Map<String, Set<String>> additionalMethodNames;
+    private static Map<String, Set<String>> additionalFields;
 
     @Option(name = "-h", aliases = "--help", usage = "Shows help")
     public boolean help;
 
     @Option(name = "-c", aliases = "--includeJavaCoreClasses", usage = "Include classes from java.* and javax.* in the report (not included by default)")
     public boolean includeJavaCoreClasses;
 
-    @Option(name = "--additionalClasses", metaVar = "FILENAME", usage = "File name for additional classes to scan")
+    @Option(name = "-C", aliases = "--additionalClasses", metaVar = "FILENAME", usage = "File name for additional classes to scan")
     public File additionalClassesFile;
 
-    @Option(name = "--onlyAdditionalClasses", depends = "--additionalClasses", usage = "Only include in the report the specified classes")
+    @Option(name = "--onlyAdditionalClasses", depends = "-C", usage = "Only include in the report the specified classes")
     public boolean onlyAdditionalClasses;
 
     @Option(name = "--onlyIncludeJenkinsClasses", usage = "Only include in the report Jenkins related classes (jenkins.*, hudson.*, etc.")
     public boolean onlyIncludeJenkinsClasses;
 
+    @Option(name = "-u", aliases = "--updateCenter", usage = "Specifies update center URL to fetch plugins from")
+    public URL updateCenterUrl;
+
+    @Option(name = "-M", aliases = "--additionalMethods", metaVar = "FILENAME", usage = "File name for additional methods to scan")
+    public File additionalMethodsFile;
+
+    @Option(name = "--onlyAdditionalMethods", depends = "-M", usage = "Only include in the report the specified methods")
+    public boolean onlyAdditionalMethods;
```

Comment:
It seems to me that the `onlyAdditional*` options could be collapsed into one. Or is there a real use case so specify some, but neither all nor none of them?

This would be a minor breaking change, but not introduce a bunch of addition options that realistically are only ever coming in a group.

Reply:
Changed to `--onlyIncludeSpecified` with the short option `-i`.